### PR TITLE
Better CLI

### PR DESCRIPTION
**Pull request by heytitle**
related to #250 

Example usages:

```
$ ./bin/pythainlp --help
usage: pythainlp [-h] command

positional arguments:
  command     [corpus|tokenization|soundex]

optional arguments:
  -h, --help  show this help message and exit

$  ./bin/pythainlp tokenization word  --text "ผมร<0e31>กประเทศไทย? สามารถ" --engine newmm
Using engine=newmm
ผม|รัก|ประเทศไทย|?| |สามารถ

$ ./bin/pythainlp tokenization syllable  --text "ผมร<0e31>กประเทศไทย? สามารถ"
Using engine=ssg
ผม~รัก~ประ~เทศ~ไทย~? ~สา~มารถ

$ ./bin/pythainlp soundex  --text "บ<0e39>รณการ" --engine lk82
บE419
```

### CODE REVIEW

**Comment by wannaphongcom** [comment]
Colab : https://colab.research.google.com/drive/1HQ8cK_SnVaGmxHCblffmZg_nsJVsbks6

File: pythainlp/cli/tokenization.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            print("No subcommand available: %s" % subcommand)
+
+    @staticmethod
+    def word(args):
+        tokens = word_tokenize(args.text)
+        print("|".join(tokens))
+
+    @staticmethod
+    def syllable(args):
+        print("kenize syllable")
```
Comment (wannaphongcom) [inline pythainlp/cli/tokenization.py]: kenize syllable ?

Comment (heytitle) [reply pythainlp/cli/tokenization.py]: sry for that. It's just temporary code.

**Comment by wannaphongcom** [comment]
Can you add Part-of-Speech (POS) tags ?

**Comment by heytitle** [comment]
Sure! My local code was outdated, so that was why I didn't implement POS. I'm going to fix it.

**Comment by heytitle** [comment]
POS added. 
```
./bin/pythainlp tagging pos --text "ผม|ไม่|กิน|เผ็ด"
```

@wannaphongcom What do you think if we slightly change the interface of commands requiring text?

In particular, it would be better if these commands receive a file instead of just a text, and write a result into another file. For example,

```
./bin/pythainlp tagging pos --input some text.txt \
 --output pos-tagged.txt \
 --engine ... 

./bin/pythainlp tokenization word --input some text.txt \
 --output tokenised-word.txt \
 --engine ... 
```
IMHO, this is the way I would use CLIs.

**Comment by wannaphongcom** [comment]
@heytitle I agree. It will greatly facilitate the use of the CLIs.

**Comment by heytitle** [comment]
Do you happen to know how many people use PyThaiNLP's CLI? 

I'm wondering whether we should create another PR for the receiving input as a file feature.

What do you think?

**Comment by wannaphongcom** [comment]
@heytitle I not know number of CLI users.  I agree.  We should create PR for the receiving input as a file feature.

**Comment by wannaphongcom** [comment]
Now, Can i merge this pull request?

**Comment by heytitle** [comment]
@wannaphongcom  yes

**Pull request merged**